symbolic-base/src/ZkFold/Base/Algebra/Polynomials/Univariate.hs

```diff
@@ -280,6 +282,13 @@ eea a b = go (a, one) (b, zero)
             where
                 (q, r) = qr x y
 
+
+diff :: Scale Integer c => Poly c -> Poly c
+diff (P cs) = P $ V.imap (\i c -> scale (fromIntegral i :: Integer) c) cs
```

Comment:
looks really Reed-Solomon-y to me, doesn't look like a function anyone else would use